lib/mwUtil.js

```diff
@@ -636,4 +644,18 @@ mwUtil.addVaryHeader = (res, vary) => {
     }
 };
 
+mwUtil.cloneRequest = (req) => {
+    if (!req) {
+        return;
```

Comment:
let's return the `req` so that we don't convert 'null' to 'undefined'. Wouldn't matter probably, just a nice style.